app/views/profiles/_show.html.erb

```diff
@@ -123,6 +123,15 @@
         <%= @profile.city %>
       </p>
     <% end %>
+    <!-- state  -->
+    <% if @profile.state.present? %>
+      <p class="">
+        <span class="font-weight-bold">
+          <%= t(:"state.#{@profile.country.downcase}") %>:
+        </span>
+        <%= t(:"states.#{@profile.country.downcase}.#{@profile.state}") %>
```

Comment:
what happens if someone adds a state but not a country?

Reply:
It's not possible, because the selectbox opens only in the case germany or austria or switzerland is selected.

Reply:
But ... thanks for the question ... there is still a bug, if someone deletes the country the state is still there, but should not.

Reply:
fixed it, so that the state is deleted if the country is deleted